src/components/v2/values/chartValuesDiff/ChartValuesView.tsx

```diff
@@ -1437,6 +1450,9 @@ function ChartValuesView({
                                     linkText={renderConnectToHelmChart()}
                                 />
                             )}
+                            {
+                              isVirtualEnvironment  && renderGeneratedDownloadManifest()
```

Comment:
we should add isVirtualEnv check inside the renderGeneratedDownloadManifest method

Reply:
Done